<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSelectInterviewDialog</class>
 <widget class="QWidget" name="QSelectInterviewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>30</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Click the &quot;Search UId&quot; button below to populate interviews based on search criteria.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="interviewTableWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Site</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>UId</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="searchPushButton">
       <property name="text">
        <string>Search UId</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QSelectInterviewDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
